wire-ios-sync-engine/Source/Public/NetworkState.swift

```diff
@@ -16,8 +16,9 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-typedef NS_ENUM(int16_t, ZMNetworkState) {
-    ZMNetworkStateOnline = 0,
-    ZMNetworkStateOffline,
-    ZMNetworkStateOnlineSynchronizing,
-};
+@objc
```

Comment:
I would have kept the prefix for objc.
```suggestion
@objc(ZMNetworkState)
```

Reply:
Good point, reverted these parts